app/Http/Controllers/Widgets/TopInterfacesController.php

```diff
@@ -54,11 +55,15 @@ public function getView(Request $request)
             ->select('port_id', 'device_id', 'ifName', 'ifDescr', 'ifAlias')
             ->groupBy('port_id', 'device_id', 'ifName', 'ifDescr', 'ifAlias')
             ->where('poll_time', '>', Carbon::now()->subMinutes($data['time_interval'])->timestamp)
+            ->where([['ignore', '=', 0], ['disabled', '=', 0]])
             ->when($data['device_group'], function ($query) use ($data) {
                 $query->inDeviceGroup($data['device_group']);
             }, function ($query) {
                 $query->has('device');
             })
+            ->when(!empty($data['hide_lo']), function ($query) {
+                $query->where('ifType', '<>', 'softwareLoopback');
```

Comment:
This should probably be made into a scope. Sounds usable

Reply:
changed :-)